metaci/api/views/testmethod_perf.py

```diff
@@ -14,12 +14,16 @@
 from metaci.plan.models import Plan
 
 from django.db import connection
+from datetime import datetime, timedelta
+
+thirty_days_ago = datetime.now() - timedelta(days=30)
```

Comment:
I was gonna object to calculating this at import time but I guess it'll be okay on Heroku with the dynos restarting daily.